DocumentedExamples/Regridding.ipynb

```diff
@@ -0,0 +1,2541 @@
+    "Then we construct the three regridders for the three different resolutions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "xESMF Regridder \n",
+       "Regridding algorithm:       bilinear \n",
+       "Weight filename:            bilinear_tracer_weights_in1degACCESSOM2_out1deg.nc \n",
+       "Reuse pre-computed weights? False \n",
+       "Input grid shape:           (300, 360) \n",
+       "Output grid shape:          (180, 360) \n",
+       "Periodic in longitude?      True"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "regridder_1degACCESSOM2_1deg = xe.Regridder(ssh_1.drop({'x', 'y'}), ds_out, 'bilinear', periodic=True, filename=\"bilinear_tracer_weights_in1degACCESSOM2_out1deg.nc\")# , reuse_weights=True)\n",
+    "regridder_1degACCESSOM2_1deg"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "xESMF Regridder \n",
+       "Regridding algorithm:       bilinear \n",
+       "Weight filename:            bilinear_tracer_weights_in025degACCESSOM2_out1deg.nc \n",
+       "Reuse pre-computed weights? False \n",
+       "Input grid shape:           (1080, 1440) \n",
+       "Output grid shape:          (180, 360) \n",
+       "Periodic in longitude?      True"
+      ]
+     },
+     "execution_count": 35,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "regridder_025degACCESSOM2_1deg = xe.Regridder(ssh_025.drop({'x', 'y'}), ds_out, 'bilinear', periodic=True, filename=\"bilinear_tracer_weights_in025degACCESSOM2_out1deg.nc\")# , reuse_weights=True)\n",
+    "regridder_025degACCESSOM2_1deg"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "xESMF Regridder \n",
+       "Regridding algorithm:       bilinear \n",
+       "Weight filename:            bilinear_tracer_weights_in010degACCESSOM2_out1deg.nc \n",
+       "Reuse pre-computed weights? False \n",
+       "Input grid shape:           (2700, 3600) \n",
+       "Output grid shape:          (180, 360) \n",
+       "Periodic in longitude?      True"
+      ]
+     },
+     "execution_count": 36,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "regridder_010degACCESSOM2_1deg = xe.Regridder(ssh_010.drop({'x', 'y'}), ds_out, 'bilinear', periodic=True, filename=\"bilinear_tracer_weights_in010degACCESSOM2_out1deg.nc\")# , reuse_weights=True)\n",
+    "regridder_010degACCESSOM2_1deg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Regridding time\n",
+    "\n",
+    "Now we use the regridders we constructed above to regrid our output."
```

Comment:
Do we need a note about all the adjustments made in this step, i.e. drop, chunk, assign_coords?